common/src/main/java/com/box/l10n/mojito/po/PoPluralRule.java

```diff
@@ -191,6 +197,12 @@ private CldrFormsToCopyOnImport(ImmutableMultimap<String, String> formMap) {
                 "1", "two",
                 "2", "few"
         )),
+        FOUR_FORMS_ONE_TWO_MANY_OTHER(ImmutableBiMap.of(
+                "0", "one",
```

Comment:
Can you put a note that this diverge from what's in https://www.gnu.org/savannah-checkouts/gnu/gettext/manual/html_node/Plural-forms.html (see doc at the very top). 

I think there are other languages that have diverged from CLDR but I kept it like in the GNU doc, I "think" for backward compatibility.